api/src/test/java/org/openmrs/module/cohort/api/dao/CohortGenericDaoTest.java

```diff
@@ -147,4 +149,14 @@ public void getByUuid_shouldReturnNullForVoidedOrRetiredCohort() {
 		
 		assertThat(dao.get(COHORT_UUID), nullValue());
 	}
+	
+	@Test
+	public void shouldVoidCohortM() {
+		CohortM cohortToVoid = dao.get(COHORT_UUID);
+		cohortToVoid.setVoided(true);
```

Comment:
Why are you setting the void properties of the cohort and then at the same time call the voidCohort service method? It is the service method that is supposed to take care of that.